packages/core/strapi/lib/commands/transfer/export.js

```diff
@@ -77,6 +77,29 @@ module.exports = async (opts) => {
   try {
     logger.log(`Starting export...`);
 
+    const progress = engine.progress.stream;
+
+    const telemetryPayload = (/* payload */) => {
+      return {
+        eventProperties: {
+          source: engine.sourceProvider.name,
+          destination: engine.destinationProvider.name,
+        },
+      };
+    };
+
+    progress.on('transfer::start', (payload = undefined) => {
```

Comment:
`= undefined` is the default's default value, no? Why do we need to specify it here? :thinking:

Reply:
The original reason was setting a default parameter thinking about typescript complaining in the future if you don't send it 😆